src/covid_model_seiir_pipeline/postprocessing/model/resampling.py

```diff
@@ -0,0 +1,47 @@
+"""Tools for resampling draw-level data."""
+from typing import Dict, List
+
+import numpy as np
+import pandas as pd
+
+from covid_model_seiir_pipeline.postprocessing.specification import ResamplingSpecification
+
+
+def build_resampling_map(deaths: pd.DataFrame, resampling_params: ResamplingSpecification):
+    cumulative_deaths = deaths.groupby(level='location_id').cumsum().reset_index()
+    cumulative_deaths['date'] = pd.to_datetime(cumulative_deaths['date'])
+    reference_date = pd.to_datetime(resampling_params.reference_date)
```

Comment:
Only new thing here is using a date from configuration instead of the last day of the forecast for draw trimming.